webpack.dev.config.js

```diff
@@ -26,7 +26,7 @@ module.exports = _.values(Merge.smart(commonConfig, {
         module: {
             rules: [
                 {
-                    test: /(.scss|.css)$/,
+                    test: /.scss$/,
```

Comment:
I am not completely sure if this has any ripple effects (I couldn't find any). Removed `css` from here as it's been moved to `webpack.common.config/js` so that react can include css files. But not sure why it was limited to paragon and font-awesome in the first place.